containerd-snapshotter/CMakeLists.txt

```diff
@@ -7,19 +7,19 @@ else()
 endif()
 
 add_custom_target(
-  cvmfs-snapshotter ALL
-  DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/cvmfs-snapshotter  
+  cvmfs_snapshotter ALL
+  DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/cvmfs_snapshotter
```

Comment:
Here we'll have a problem because the source directory is still named `cvmfs_snapshotter`. How about we simply rename it to `snapshotter` and adjust the cmake files accordingly.